services/blue/src/api/DatabaseAccess/DbConnectionHolder.cs

```diff
@@ -70,13 +70,13 @@ public IDbCommand CreateCommand()
 
         public async ValueTask DisposeAsync()
```

Comment:
If we implement `IAsyncDisposable` we should also implement `IDisposable` as per  https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-disposeasync. See `BlackClient` for reference.

Reply:
already implemented:
```csharp
public void Dispose() => _connection.Dispose();
```